src/test/java/StringTest.java

```diff
@@ -0,0 +1,61 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+
+    @DisplayName("1,2를 ','로 split 했을 때 1과 2로 잘 분리되는지 확인한다.")
+    @Test
+    void splitTest() {
+        // given
+        String str = "1,2";
+
+        // when
+        String[] sut = str.split(",");
+
+        // then
+        assertAll(() -> assertThat(sut).contains("1", "2"), () -> assertThat(sut).containsExactly("1", "2"));
```

Comment:
`containsExactly` 검증만으로 `contains`를 포함하고 있으니 `contains` 검증은 제거해도 좋습니다.
```suggestion
        assertThat(sut).containsExactly("1", "2");
```